src/main/java/uk/gov/hmcts/darts/audio/service/impl/MediaRequestServiceImpl.java

```diff
@@ -56,6 +58,12 @@ public List<MediaRequestEntity> getMediaRequestsByStatus(AudioRequestStatus stat
         return mediaRequestRepository.findByStatusOrderByCreatedDateTimeAsc(status);
     }
 
+    @Override
+    @Transactional(propagation = Propagation.SUPPORTS)
+    public Optional<MediaRequestEntity> getOldestMediaRequestByStatus(AudioRequestStatus status) {
```

Comment:
I think this is a good change, but consider removing the `getMediaRequestsByStatus()` set of methods as these are now redundant with this change.